config/manifests/bases/clusterserviceversion.yaml.in

```diff
@@ -31,6 +31,16 @@ spec:
       kind: ReclaimSpaceJob
       name: reclaimspacejobs.csiaddons.openshift.io
       version: v1alpha1
+    - description: EncryptionKeyRotationCronJob is the Schema for the encryptionkeyrotationcronjobs API
```

Comment:
@black-dragon74 can you please open an issue for this one, In our CI we should identify this kind of inconsistency

Reply:
Done: https://github.com/csi-addons/kubernetes-csi-addons/issues/670

Reply:
can you please add the volumegroupreplication CRDs as well, it looks like that is also missing.

Reply:
Updated commit desc as well :)